frontend/website/src/HeroSection.re

```diff
@@ -0,0 +1,131 @@
+open Css;
+open Style;
```

Comment:
I wonder if we should avoid doing this at the top level, those modules define a lot of identifiers